### PR TITLE
Update framework.rst

### PR DESCRIPTION
**Pull request by flo-sch**
Correct documentation for the `config.assets` node.

Basically :

```
framework:
    templating:
        assets_version: ~
        assets_version_format: ~
        assets_base_url: ~
        packages: ~
```

Should now be configurated as :

```
framework:
    assets:
        version: ~
        version_format: ~
        base_url: ~
        packages: ~
```

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -858,10 +859,136 @@ setting the value to ``null``:
 templating
 ~~~~~~~~~~
 
-.. _reference-framework-assets-version:
+hinclude_default_template
```
Comment (OskarStark) [inline reference/configuration/framework.rst]: is the ``h`` correct at the beginning?

Comment (OskarStark) [inline reference/configuration/framework.rst]: ah yes it is :+1:

File: reference/configuration/framework.rst [hunk 2]
```diff
@@ -858,10 +859,136 @@ setting the value to ``null``:
...
+        <!-- app/config/config.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+
```
Comment (OskarStark) [inline reference/configuration/framework.rst]: why all of these blank lines?

**Comment by OskarStark** [comment]
Good job :+1:

File: reference/configuration/framework.rst (hunk 2 above)
Comment (Flo-Schield-Bobby) [inline reference/configuration/framework.rst]: Oh, I just copy-pasted the XML sample from another section of the documentation, actually I never use any XML config...

Why blank lines ? I guess it doesn't really affect the behaviour of the XML parser, but gains in visibility ?

**Comment by Flo-Schield-Bobby** [comment]
Thanks :)

File: reference/configuration/framework.rst (hunk 1 above)
Comment (Flo-Schield-Bobby) [inline reference/configuration/framework.rst]: Yes, it is related to the `hinclude` javascript library :
http://mnot.github.io/hinclude/

File: reference/configuration/framework.rst (hunk 2 above)
Comment (OskarStark) [inline reference/configuration/framework.rst]: no problem :smile:

File: reference/configuration/framework.rst (hunk 1 above)
Comment (OskarStark) [inline reference/configuration/framework.rst]: :+1:

**Comment by OskarStark** [comment]
Status: Reviewed

**Comment by xabbuh** [comment]
@flo-sch I am sorry that we missed your PR and did not respond in time. It looks like that in the meantime we merged other pull requests that made the same changes. Sadly that means that there is nothing left from your PR that we can merge. Again please apologise our lack of feedback and please come back with more suggestions to improve the docs soon.

**Pull request closed without merging**